Min-Max_Guide.md

```diff
@@ -308,45 +311,44 @@ Another alternative is to exploit RNG to farm clay, but this is illegal in our r
-  - By planting additional crops beyond the 15 initial Parsnip Seeds, crows have a chance to eat one or more crops every day. However, the extra value from planting the Mixed Seeds makes this worth it. (Having random Cauliflower and Potato from Mixed Seeds allows you to skip buying them for the Spring Crops bundle later on.)
-- Eat all Spring Onions. (They are near-worthless to sell.)
+  - Crows can eat one or more crops every day by planting additional crops beyond the 15 initial Parsnip Seeds. However, planting the Mixed Seeds makes this worth the extra value. (Having random Cauliflower and Potato from Mixed Seeds allows you to skip buying them for the Spring Crops bundle later.)
 - Chop wood until around 4-6 energy left.
-- Kill weeds + grass until around 1:00 AM to 1:30 AM, depending on how many Mixed Seeds you have. (You want to hoe and plant and water all of the Mixed Seeds before 2 AM and passing out.)
+- Kill weeds + grass until around 1:00 AM to 1:30 AM, depending on how many Mixed Seeds you have. (You want to hoe, plant, and water all the Mixed Seeds before 2 AM and pass out.)
 - Head back to your house and ship everything except:
   - Stone
   - Wood
   - Coal
   - 20 Fiber
 - Note that:
-  - You should sell all foraging items (except for Spring Onions). Worry about the Community Center quest later on, because we need as much money as possible for tomororw.
-  - Normally, we eat Leeks, Dandelions, and food from the garbage cans instead of selling them, since they have a low GPE. However, today as an exception to the rule, since we need money very badly.
+  - You should sell all foraging items (except for Spring Onions). Worry about the Community Center quest later because we need as much money as possible tomorrow.
+  - Normally, we eat Leeks, Dandelions, and food from garbage cans instead of selling them since they have a low GPA. However, today is an exception since we need money badly.
 - Hoe + plant + water any remaining Mixed Seeds.
   - Note that you can work past 0 energy and become exhausted with no penalty carrying over to Spring 2 (because you will level up foraging).
-  - If one of the mixed seeds is not a Cauliflower (check with a mod or the wiki) you'll need to delay handing in the initial bundles and shipping any additional gold for the vault originally scheduled for Spring 15 until Spring 17.
-  - This adds a bit of time pressure to clearing the mine due to the lack of Minecarts you'd otherwise have for 2 days but this is very unlikely and doesn't change the route.
+  - If one of the mixed seeds is not a Cauliflower (check with a mod or the wiki), you'll need to delay handing in the initial bundles and shipping any additional gold for the vault originally scheduled for Spring 15 until Spring 17.
+  - This adds a bit of time pressure to clearing the mine due to the lack of Minecarts you'd otherwise have for two days, but this is very unlikely and doesn't change the route.
```

Comment:
why are we changing 2 to two

Reply:
Grammarly preference. 

https://www.grammarly.com/blog/when-to-spell-out-numbers/#:~:text=It%20is%20generally%20best%20to,should%20be%20expressing%20numbers%20consistently.

Fine to change it back if you prefer.